pkg/compiler/types.go

```diff
@@ -830,6 +830,28 @@ var typeFmtFormat = &typeArg{
 	Kind:  kindIdent,
 }
 
+// typeCompressed is used for compressed data.
+var typeCompressed = &typeDesc{
+	Names:     []string{"compressed"},
+	CantBeOpt: true,
+	CantBeOut: true,
```

Comment:
This allows images to be outputs from the kernel. Do we want this?

Reply:
(as above with `CantBeOpt`)